pcdet/datasets/augmentor/database_sampler.py

```diff
@@ -3,18 +3,27 @@
 import os
 import copy
 import numpy as np
+from skimage import io
+import torch
 import SharedArray
 import torch.distributed as dist
 
 from ...ops.iou3d_nms import iou3d_nms_utils
-from ...utils import box_utils, common_utils
-
+from ...utils import box_utils, common_utils, calibration_kitti
+from pcdet.datasets.kitti.kitti_object_eval_python import kitti_common
 
 class DataBaseSampler(object):
     def __init__(self, root_path, sampler_cfg, class_names, logger=None):
         self.root_path = root_path
         self.class_names = class_names
         self.sampler_cfg = sampler_cfg
+        self.aug_with_img = sampler_cfg.get('AUG_WITH_IMAGE', False)
+        self.joint_sample = sampler_cfg.get('JOINT_SAMPLE', False)
```

Comment:
Can we remove these image-related configs and use them directly at their corresponding positions?  
like `if self.sampler_cfg.get('JOINT_SAMPLE', False)`. 
Since these configs are not generally utilized configs for this class and are only used at very few positions.

Reply:
Yes. I remove these configs and use them directly.

https://github.com/yukang2017/OpenPCDet/blob/a1bd2d7bc3c093dc8a934f43b5f7a2993068d8e4/pcdet/datasets/augmentor/database_sampler.py#L21